src/main/java/uk/gov/laa/ccms/caab/builders/ApplicationBuilder.java

```diff
@@ -113,7 +113,7 @@ public ApplicationBuilder provider(UserDetail user) {
    * @return The builder instance.
    */
   public ApplicationBuilder client(ClientDetail clientInformation) {
```

Comment:
shouldn't we be `finalising` parameters?

Reply:
amended